cypress/e2e/signIn.cy.js

```diff
@@ -2,10 +2,29 @@
 
 describe('Sign In page', () => {
   beforeEach(() => {
+    cy.visit('/login');
+  });
+
+  const username = 'tomsmith';
+  const password = 'SuperSecretPassword!';
+
+  it('should login user with valid credentials', () => {
+    cy.loginUser(username, password);
+
+    cy.contains('h2', 'Secure Area');
+
+    cy.checkAuthByHref('Logout');
+  });
+
+  it('assert validation error for login with invalid username', () => {
+    cy.loginUser(`${username}1`, password);
```

Comment:
```suggestion
    cy.loginUser(`${username}1`, password);
```
create a separate variable invalidUsername for this ${username}1